css/components.css

```diff
@@ -0,0 +1,517 @@
+/* button Started */
+.avtr,.avtr-clr,.sm-avtr-clr,.avtr-sq,.sm-avtr-sq{
```

Comment:
## Naming Conventions
1. Avoid short forms while naming classes, giving proper names help in reading the code better.
2. Replace `avtr` with `avatar`.